applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -33,6 +33,7 @@ Syntax
   * ``2`` - Start FOTA for modem delta update.
   * ``3`` - Start FOTA for full modem update.
     Can only be used when the :file:`overlay-full_fota.conf` configuration file is used.
+    Not supported on the :ref:`Thingy:91 <thingy91_ug_intro>`` as it lacks an external flash to store the firmware image.
```

Comment:
```suggestion

    Not supported on the :ref:`Thingy:91 <thingy91_ug_intro>` as it lacks an external flash to store the firmware image.
 
```
Extra backtick is what's breaking the build.
I added the empty lines to make it stand out a bit more.